backend/index.js

```diff
@@ -19,4 +20,36 @@ app.get("/", (req, res) => {
   res.send("Hello world!");
 });
 
+// #1 - get all events
+app.get("/api/events", async (req, res) => {
+  try {
+    const events = await Event.find({});
+    res.send(events);
+  } catch (error) {
+    res.status(500).send(error.message);
+    console.log(`error is ${error.message}`);
+  }
+});
+
+// #2 - add an event
+app.post("/api/recipe", async (req, res) => {
+  try {
+    const { title, Location, start, end, slots, notes, Volunteers } = req.body;
+    let event = new Event({
+      title,
+      Location,
+      start,
+      end,
+      slots,
+      notes,
+      Volunteers,
```

Comment:
We can initialize volunteers to an empty array since there won't be any volunteers when we first make an event

Reply:
Should I do that in the model?